src/components/Elements/Dialog.js

```diff
@@ -34,7 +36,24 @@ const Dialog = (props) => {
     'error': 'neon-coral',
   }
 
+  function createMarkup() {
```

Comment:
Like I said before, I'd rather we used a bundle release notes for this to avoid the possibility of malicious code. But at least there are some precautions in place.